harness/tests/experiment/keras/test_tf_keras_trial.py

```diff
@@ -368,82 +368,6 @@ def make_workloads() -> workload.Stream:
         controller.run()
 
 
-@pytest.mark.tensorflow
-def test_cifar10(tmp_path: Path):
-    """
-    Make sure each example:
-     - trains
-     - validates
-     - checkpoints
-     - can load from checkpoint
-    """
-    checkpoint_dir = str(tmp_path.joinpath("checkpoint"))
-    latest_checkpoint = None
-    steps_completed = 0
-
-    def make_workloads_1() -> workload.Stream:
-        """
-        Train one batch, validate one batch, checkpoint.
-        """
-        trainer = utils.TrainAndValidate()
-        yield from trainer.send(steps=2, validation_freq=2, scheduling_unit=1)
-
-        interceptor = workload.WorkloadResponseInterceptor()
-        yield from interceptor.send(workload.checkpoint_workload())
-        nonlocal latest_checkpoint, steps_completed
-        latest_checkpoint = interceptor.metrics_result()["uuid"]
-        steps_completed = trainer.get_steps_completed()
-
-    example_path = utils.cv_examples_path("cifar10_tf_keras/model_def.py")
```

Comment:
Does swapping `iris` in-place not work here?

Reply:
There seemed to be other unit tests which do exercise `TFKerasTrials`, so I thought there's no point in keeping this one if we have an e2e for it anyway. I can bring it back if you think it's valuable.

Reply:
The `test-e2e-gpu-parallel` workflow only applies to main branch. To run this on dev branches, it has to be manually run in `test-e2e-longrunning`. I think keeping a unit test around for it is probably a good idea.

Reply:
See: https://github.com/determined-ai/determined/blob/48874503af2b4dcc69b810721e1d5b7c2d34dfaa/.circleci/real_config.yml#L2742
and any other hits in that file containing `mark: ["parallel"]`